Uno.Gallery/Uno.Gallery.Shared/Extensions/XamlDisplayExtensions.cs

```diff
@@ -80,6 +80,32 @@ public static class XamlDisplayExtensions
 
 		#endregion
 
+		#region Property: ShowXaml
+
+		public static DependencyProperty ShowXamlProperty { get; } = DependencyProperty.RegisterAttached(
+			"ShowXaml",
+			typeof(bool),
+			typeof(XamlDisplayExtensions),
+			new PropertyMetadata(default, (d, e) => d.Maybe<XamlDisplay>(control => OnShowXamlChanged(control, (bool)e.NewValue))));
```

Comment:
```suggestion
			new PropertyMetadata(default(bool), (d, e) => d.Maybe<XamlDisplay>(control => OnShowXamlChanged(control, (bool)e.NewValue))));
```
otherwise you are passing `null` as default value

Reply:
Really? doesn't the `default` literal infer the type and default to False? https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/default#default-literal

Reply:
yes, but it depends on the context, which, in this case, it is an `object`:
```cs
public PropertyMetadata(object defaultValue, PropertyChangedCallback propertyChangedCallback);
```